src/content/plugins/split-chunks-plugin.md

```diff
@@ -145,38 +146,38 @@ Maximum number of parallel requests at an entry point.
 
 ### `splitChunks.minChunks`
 
-`number`
+`number = 1`
 
 Minimum number of chunks that must share a module before splitting.
 
 ### `splitChunks.minSize`
 
-`number`
+`number = 0`
 
 Minimum size, in bytes, for a chunk to be generated.
 
 ### `splitChunks.enforceSizeThreshold`
 
-#### `splitChunks.cacheGroups.{cacheGroup}.enforceSizeThreshold`
-
 `number = 50000`
 
+#### `splitChunks.cacheGroups.{cacheGroup}.enforceSizeThreshold`
+
 Size threshold at which splitting is enforced and other restrictions (minRemainingSize, maxAsyncRequests, maxInitialRequests) are ignored.
 
 
 ### `splitChunks.minRemainingSize`
 
-#### `splitChunks.cacheGroups.{cacheGroup}.minRemainingSize`
+`number = 0`
```

Comment:
Moving it here means you have to provide a separate type mention after `#### splitChunks.cacheGroups.{cacheGroup}.minRemainingSize`

Reply:
Okay, so I simply should keep the position below `{cacheGroup}.minRemainingSize`?

Reply:
Done in 1958bd5